extensions/wikia/PhalanxII/PhalanxSpecialController.class.php

```diff
@@ -106,7 +106,7 @@ public function main() {
 
 		// VSTF should not be allowed to block emails in Phalanx
 		$showEmailBlock = $this->wg->User->isAllowed( 'phalanxemailblock' );
-		$blockTypes  = Phalanx::getAllTypeNames();
+		$blockTypes  = Phalanx::getSupportedTypeNames();
```

Comment:
We got rid of `128 => 'cookie'` as a block type back in early 2016 but that bit value still lives on in some of our stats.  Make sure there is a way to return the types we use while still having all the types that exist available for stats reporting.